packages/localize/examples/runtime-ts/package.json

```diff
@@ -5,9 +5,18 @@
   "private": true,
   "type": "module",
   "scripts": {
-    "build": "rm -rf lib/* && lit-localize build && tsc",
+    "build": "wireit",
     "serve": "web-dev-server --node-resolve"
   },
+  "wireit": {
+    "build": {
+      "command": "rm -rf lib/* && lit-localize build && tsc",
+      "dependencies": [
+        "../../../localize:build",
+        "../../../localize-tools:build"
+      ]
+    }
```

Comment:
Maybe not worth the effort but should this also have files and output specified like mentioned above? Including `lib` in output should also remove the need for the `rm -rf`.